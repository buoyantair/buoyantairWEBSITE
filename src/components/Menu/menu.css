.menu {
  grid-row: 3;
  display: grid;
  grid-template-columns: 1fr;
  margin-left: 40px;
}

.menu-item {
  font-size: 14px;
  font-weight: bold;
  color: black;
  text-decoration: none;
  height: 20px;
}
.menu-item:hover,
.menu-item:active,
.menu-item:focus {
  color: black;
  text-decoration: none;
  outline: none;
}

.menu-item:before {
  content: '';
  display: block;
  background: black;
  width: 30px;
  height: 2px;
  position: relative;
  left: -40px;
  top: 10px;
}
