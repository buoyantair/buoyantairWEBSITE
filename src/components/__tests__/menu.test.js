import React from "react"
import renderer from "react-test-renderer"

import Menu from "../menu"

describe("Menu", () => {
  const emptyMenu = []
  const menuWithItems = [
    {
      name: 'home',
      path: '/'
    },
    {
      name: 'about',
      path: '/about'
    }
  ]
  it("renders empty menu correctly", () => {
    const tree = renderer
      .create(<Menu menuItems={emptyMenu} />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })

  it("renders menu with items correctly", () => {
    const tree = renderer
      .create(<Menu menuItems={menuWithItems} />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})