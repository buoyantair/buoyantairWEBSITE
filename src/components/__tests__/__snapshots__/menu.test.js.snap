// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu renders empty menu correctly 1`] = `<nav />`;

exports[`Menu renders menu with items correctly 1`] = `
<nav>
  <a
    href="/"
  >
    home
  </a>
  <a
    href="/about"
  >
    about
  </a>
</nav>
`;
